### PR TITLE
Add: Seeed Studio MR60FDA2 mmWave Sensor document

### PR DESCRIPTION
**Pull request by limengdu**
## Description:

**Related issue (if applicable):** fixes <link to issue>

**Pull request in [esphome](https://github.com/esphome/esphome) with YAML changes (if applicable):** esphome/esphome#7576

## Checklist:

  - [x] I am merging into `next` because this is new documentation that has a matching pull-request in [esphome](https://github.com/esphome/esphome) as linked above.  
    or
  - [ ] I am merging into `current` because this is a fix, change and/or adjustment in the current documentation and is not for a new component or feature.

  - [x] Link added in `/index.rst` when creating new documents for new components or cookbook.

### CODE REVIEW

File: components/seeed_mr60fda2.rst [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+    text_sensor:
+      - platform: seeed_mr60fda2
+        is_fall:
+          name: "Falling Information"
+
+Configuration variables:
+************************
+
+- **is_fall** (*Optional*): Fall status indication.
+  All options from :ref:`Text Sensor <config-text_sensor>`.
```
Comment (jesserockz) [inline components/seeed_mr60fda2.rst]: This has moved to `binary_sensor`

File: components/seeed_mr60fda2.rst [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+The ``seeed_mr60fda2`` button allows you to perform actions.
+
+.. code-block:: yaml
+
+    button:
+      - platform: seeed_mr60fda2
+        get_radar_parameters:
+          name: "Get Radar Parameters"
+        reset_radar:
```
Comment (jesserockz) [inline components/seeed_mr60fda2.rst]:
```suggestion
        factory_reset:
```

File: components/seeed_mr60fda2.rst [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+          name: "Reset"
+
+Configuration variables:
+************************
+
+- **reset_radar**: Restore all radar settings to factory parameters. All options from :ref:`Button <config-button>`.
```
Comment (jesserockz) [inline components/seeed_mr60fda2.rst]:
```suggestion
- **factory_reset**: Restore all radar settings to factory parameters. All options from :ref:`Button <config-button>`.
```

File: components/seeed_mr60fda2.rst [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+============================================================
+
+.. seo::
+    :description: Instructions for setting up Seeed Studio MR60FDA2 60GHz mmWave Fall Detection Sensor Kit.
+    :image: seeed_mr60fda2.jpg
+
+Component/Hub
+-------------
+
+The ``seeed_mr60fda2`` platform allows you to use Seeed Studio MR60FDA2 60GHz mmWave Fall Detection Sensor Kit with XIAO ESP32C6 (`Product Page <https://www.seeedstudio.com/MR60FDA2-60GHz-mmWave-Sensor-Fall-Detection-Module-p-5946.html>`__) with ESPHome.
```
Comment (kbx81) [inline components/seeed_mr60fda2.rst]:
```suggestion
The ``seeed_mr60fda2`` platform allows you to use Seeed Studio's MR60FDA2 60GHz mmWave Fall Detection Sensor Kit with XIAO ESP32C6 (`Product Page <https://www.seeedstudio.com/MR60FDA2-60GHz-mmWave-Sensor-Fall-Detection-Module-p-5946.html>`__) with ESPHome.
```

File: components/seeed_mr60fda2.rst [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+
+The :ref:`UART <uart>` is required to be set up in your configuration for this sensor to work, ``parity`` and ``stop_bits`` **must be** respectively ``NONE`` and ``1``.
+You can use the ESP32 software or hardware serial to use this MR60FDA2, its default baud rate is 115200.
```
Comment (kbx81) [inline components/seeed_mr60fda2.rst]:
```suggestion
You can use the ESP32 software or hardware (recommended) serial to use the MR60FDA2; its default baud rate is 115200.
```

File: components/seeed_mr60fda2.rst [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+************************
+
+- **uart_id** (*Optional*, :ref:`config-id`): Manually specify the ID of the :ref:`UART Component <uart>` if you want
+  to use multiple UART buses.
+- **id** (*Optional*, :ref:`config-id`): Manually specify the ID for this :doc:`seeed_mr60fda2` component if you need multiple components.
+
+Binary Sensor
+-------------
+
+The ``seeed_mr60fda2`` binary sensor allows you to perform different measurements.
```
Comment (kbx81) [inline components/seeed_mr60fda2.rst]:
```suggestion
The ``seeed_mr60fda2`` binary sensor allows you to determine the presence of a human.
```

File: components/seeed_mr60fda2.rst [hunk 7]
```diff
@@ -0,0 +1,126 @@
...
+
+    binary_sensor:
+      - platform: seeed_mr60fda2
+        people_exist:
+          name: "Person Information"
+
+Configuration variables:
+************************
+
+- **people_exist** (*Optional*): If true target detected, otherwise no target.
```
Comment (kbx81) [inline components/seeed_mr60fda2.rst]:
```suggestion
- **people_exist** (*Optional*): If true when target (person) is detected.
```

**Review by jesserockz — CHANGES_REQUESTED** [review]

File: components/seeed_mr60fda2.rst [hunk 8]
```diff
@@ -0,0 +1,126 @@
...
+
+- **reset_radar**: Restore all radar settings to factory parameters. All options from :ref:`Button <config-button>`.
+- **get_radar_parameters**: Get all the current setup parameters of the radar.
+  All options from :ref:`Button <config-button>`.
+
+
+Text Sensor
+-----------
+
+The ``seeed_mr60fda2`` text sensor allows you to get information about your device.
```
Comment (kbx81) [inline components/seeed_mr60fda2.rst]:
```suggestion
The ``seeed_mr60fda2`` text sensor allows you to determine information about a detected fall.
```

File: components/seeed_mr60fda2.rst [hunk 9]
```diff
@@ -0,0 +1,126 @@
...
+Configuration variables:
+************************
+
+- **is_fall** (*Optional*): Fall status indication.
+  All options from :ref:`Text Sensor <config-text_sensor>`.
+
+Select
+-----------
+
+The ``seeed_mr60fda2`` select allows you to control the configuration.
```
Comment (kbx81) [inline components/seeed_mr60fda2.rst]:
```suggestion
Select
------

The ``seeed_mr60fda2`` select allows you to control the sensor's configuration.
```

**Review by kbx81 — DISMISSED** [review]
A few minor copy fixes/corrections 🙂

File: components/seeed_mr60fda2.rst (hunk 5 above)
Comment (jesserockz) [reply components/seeed_mr60fda2.rst]: I actually dont think hardware vs software is needed here. ESP32 only has hardware uart anyway

File: components/seeed_mr60fda2.rst (hunk 7 above)
Comment (jesserockz) [reply components/seeed_mr60fda2.rst]: Maybe this sensor key should be `has_target` in line with other presense sensors?

File: components/seeed_mr60fda2.rst (hunk 4 above)
Comment (limengdu) [reply components/seeed_mr60fda2.rst]: But, that's the name of our product.

**Review by jesserockz — APPROVED** [review]

**Pull request merged**